Makefile

```diff
@@ -32,13 +32,9 @@ apply:
 
 ## Runs govet and gofmt against all packages.
 .PHONY: check-style
-check-style: webapp/.npminstall gofmt govet golint
+check-style: webapp/.npminstall gofmt govet
```

Comment:
OT: should we take golint out of the template? I am not aware of 1 plugin that we have that passes without tweaking the checks.

Reply:
I personally removed it because I was having issues with my `golint` in my IDE and wanted to iterate quickly for the hackathon. @jfrerich has made the necessary changes to pass the `golint` now. I will put it back in the Makefile.

@levb Re: the starter template, I would add that we are not using `golint` in the server so was suprised that it was included here.